rust_src/src/fonts.rs

```diff
@@ -143,4 +147,45 @@ pub fn find_font(spec: LispObject, frame: LispObject) -> LispObject {
     }
 }
 
+/// Return a font-object for displaying a character at POSITION.
+/// Optional second arg WINDOW, if non-nil, is a window displaying
+/// the current buffer.  It defaults to the currently selected window.
+/// Optional third arg STRING, if non-nil, is a string containing the target
+/// character at index specified by POSITION.
+#[lisp_fn(min = "1", c_name = "font_at", name = "font-at")]
+pub fn font_at_lisp(
+    position: LispObject,
+    window: LispWindowLiveOrSelected,
+    string: LispObject,
+) -> LispObject {
+    let mut w: LispWindowRef = window.into();
+    let cur_buf = ThreadState::current_buffer();
+
+    let pos = if let Some(s) = string.as_string() {
+        let pos = position.as_fixnum_or_error() as isize;
```

Comment:
@nickdrozd has been migrating us to `into()` and `from()` calls.
```suggestion
        let pos = EmacsInt::from(position) as isize;
```